backend/src/swa-tracker/swa-tracker-wrapper.ts

```diff
@@ -0,0 +1,66 @@
+import { SWATracker } from "@sap/swa-for-sapbas-vsx";
+import { getLogger } from "../logger/logger-wrapper";
+
+const YEOMAN_UI = "Application Wizard";
+
+// Event types used by Application Wizard
+export const EVENT_TYPES = {
+	PROJECT_GENERATION_STARTED: "Project generation started",
+	PROJECT_GENERATED_SUCCESSFULLY: "Project generated successfully",
+	PROJECT_GENERATION_FAILED: "Project generation failed",
+	EXPLORE_AND_INSTALL_GENERATORS_LINK: "Explore and Install Generators link",
+	PREVIOUS_STEP: "One of the previous steps is clicked"
+}; 
+
+/**
+ * A Simple Wrapper to hold the state of our "singleton" (per extension) SWATracker
+ * implementation.
+ */
+
+export const ERROR_ANALYTICS_TRACKER_NOT_INITIALIZED = 'Analytics tracker has not yet been initialized!';
+
+let swaTracker: SWATracker;
+
+function isInitialized(): boolean {
+  return (swaTracker !== undefined ) ? true : false;
```

Comment:
return (swaTracker !== undefined );

Reply:
This is copy-paste from logger wrapper.
I prefer we have a consistent code.
Instead of this I can no problem to change. I think this is same.